Week0 - faces

```diff
@@ -0,0 +1,18 @@
+def convert(emoji):
+    emoji = emoji.replace(":)", "🙂")
+    emoji = emoji.replace(":(", "🙁")
+    return emoji
+def main():
+    inp = input()
+    print(convert(inp))
+main()
+
+# In a file called faces.py, implement a function called convert that accepts a str as input and 
+# returns that same input with any :) converted to 🙂 (otherwise known as a slightly smiling face)
+# and any :( converted to 🙁 (otherwise known as a slightly frowning face). All other text should 
+# be returned unchanged.
+
+# Then, in that same file, implement a function called main that prompts the user for input, calls 
```

Comment:
An improvement would be printing a message that the text is unchanged. printing within convert is trivial, can you do it so that you print in main?